frontend/components/RenderTask.tsx

```diff
@@ -0,0 +1,19 @@
+function RenderTask(props: {
+  id: number;
+  title: string;
+  completed: boolean;
+  func: Function;
```

Comment:
- as a general rule, try not to use abbreviations when naming, it saves few characters and just introduces additional work for the reader of the code to understand
- more importantly regarding the name `func` or even `function` - in line with previous comments, name this after what it does, not what it is.. A conventional name would be: 'changeHandler'